### PR TITLE
Add language parameter

### PR DESCRIPTION
**Pull request by hollandmatt**
Adds support for localizing the map as described: https://developers.google.com/maps/documentation/javascript/examples/map-language

### CODE REVIEW

File: test/GoogleMaps.js [hunk 1]
```diff
@@ -108,6 +108,13 @@
 				expect(GoogleMapsLoader.createUrl()).to.be.equal(baseUrl + '?callback=' + cb + '&sensor=false&channel=abcdefghijkl');
 			});
 
+			it('should create url with language', function() {
+				GoogleMapsLoader.CHANNEL = undefined;
```
Comment (sakren) [inline test/GoogleMaps.js]: Please, can you remove this line? Or is it needed?

Comment (hollandmatt) [inline test/GoogleMaps.js]: Without it, the setting for CHANNEL from the previous test was still applied. Do you think I missed something elsewhere?

Comment (sakren) [inline test/GoogleMaps.js]: No you didn't, sorry.. It's actualy a bug..

CHANNEL variable is not cleared when `release` method is [called](https://github.com/Carrooi/Node-GoogleMaps/blob/develop/lib/Google.js#L133-L137).

Later today I'll fix that, may I ask you than to rebase and update this PR please? I'll let you know when it's done.

Comment (hollandmatt) [inline test/GoogleMaps.js]: No problem :+1:

Comment (sakren) [inline test/GoogleMaps.js]: So I made the update 96205dfd42f3d1fc15bc49ba5399f36c0c200e01 :wink:

**Comment by hollandmatt** [comment]
@sakren That's all up to date now!

**Comment by hollandmatt** [comment]
Hey @sakren Any more feedback?

**Comment by sakren** [comment]
Hello, sorry I forgot..

Unfortenately I don't know what to do now. You see there should be only one (your) commit. When I was talking about rebase, I meant rebase to develop + force push and squash (if it was needed). But now I don't know if already rebased branch can be rebased again (onto same branch as before - develop) and force pushed.. Also I don't want to bother you even more...

So maybe the most simple way would be to create a new pull request which I can immediatelly merge?

I'm sorry, that I'm so tiresome :stuck_out_tongue:

**Comment by hollandmatt** [comment]
Oh, I see now! Sorry, I misunderstood you there - I'll do a new PR and close this one.

**Pull request closed without merging**